storm-core/src/clj/backtype/storm/daemon/executor.clj

```diff
@@ -685,7 +685,7 @@
           0))
       :kill-fn (:report-error-and-die executor-data)
       :factory? true
-      :thread-name component-id)]))
+      :thread-name (str component-id "-executor" (:executor-id executor-data)))]))
```

Comment:
Thanks @HeartSaVioR 
Currently the executor send queue thread has name like this:
 "Thread-25-disruptor-executor[25 25]-send-queue"

I take similar format to above to be consistent, now we will have executor thread name as:
"Thread-26-spout-executor[25 25]"